Content.Server/GameObjects/Components/Timing/UseDelayComponent.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+using Robust.Shared.Timers;
+
+namespace Content.Server.GameObjects.Components.Timing
+{
+    /// <summary>
+    /// Timer that creates a cooldown each time an object is activated/used
+    /// </summary>
+    [RegisterComponent]
+    public class UseDelayComponent : Component
+    {
+        public override string Name => "UseDelay";
+
+        /// <summary>
+        /// The time, in milliseconds, between an object's use and when it can be used again
+        /// </summary>
+        public int _delay;
+        public bool _activeDelay;
```

Comment:
Both of these should be properties. They should be pascal case and `ActiveDelay` should probably be `private set;` too.

Reply:
For some reason the standard seems to be camelcase in multiple other components.. might fix that in another PR.